### PR TITLE
Health cleanup 

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Removes a large amount of unused and or old code in the old health folder.
Renames bodyPartModification to Organ.
Moves some enums to a new file called HealthEnums in HealthV2.
Removes the v3 player prefab.

as consequence, simple mobs will no longer be affected by radiation and blood loss

### CODE REVIEW

**Comment by ThatDan123** [comment]
image

Nice

**Comment by corp-0** [comment]
I'm not sure if compiling enums into a single file is the move

File: UnityProject/Assets/Scripts/UI/Core/Action/ItemActionButton.cs [hunk 1]
```diff
@@ -70,14 +70,6 @@ public void OnInventoryMoveClient(ClientInventoryMove info)
 			ClientSetActionButtonVisibility(shouldShow);
 
 			if (PlayerManager.LocalPlayerScript == null || PlayerManager.LocalPlayerScript.playerHealth == null) return;
-			if (shouldShow)
-			{
-				PlayerManager.LocalPlayerScript.playerHealth.OnDeathNotifyEvent += OnDeath;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Core/Action/ItemActionButton.cs:75]: why remove this?

Comment (Aranclanos) [reply UnityProject/Assets/Scripts/UI/Core/Action/ItemActionButton.cs:75]: its not used

File: UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs [hunk 2]
```diff
@@ -9,8 +9,8 @@
 namespace HealthV2
 {
 	/// <summary>
-	/// A part of a body. Can be external, such as a limb, or internal like an organ.
-	/// Body parts can also contain other body parts, eg the 'brain body part' contained in the 'head body part'.
+	/// An external part of a body. Limbs, chest, head, wings, etc.
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs]: this is incorrect, since organ still use BodyPart

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs]: such as the game object heart and such

Comment (Aranclanos) [reply UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs]: reverted

**Pull request merged**